src/content/docs/style-guide/writing-docs/processes-procedures/create-edit-content.mdx

```diff
@@ -41,11 +41,16 @@ A member of the Docs team will review your pull request and comment with any fee
 
 If you'd like to go deeper with development, see our [Contributors guide](https://github.com/newrelic/docs-website/blob/develop/CONTRIBUTING.md#editing-existing-pages) for information on how to fork our site, build it locally, and submit pull requests.
 
-## Clone (copy) an existing doc [#clone-doc]
+## Edit an existing branch [#one-doc]
 
-Once you've cloned the `docs-website` repository, use your text editor to copy an existing doc. Rename and edit the copy and then save it as a new doc. Your cloned doc automatically inherits the original doc's frontmatter content. Make sure to change that, too.
+If you're a contributor who is collaborating with a writer who has already started some work you'd like to contribute to, you can also do this in GitHub. You'll need to create a fork of the docs repository, add your changes in the fork, and then create a PR to get your changes published in the original branch.
```

Comment:
Can we abstract this a little bit? Maybe make this a bit more procedural/simple?

Something like:

"Do you want to contribute to an existing PR? On GitHub, follow these steps:

1. Create a fork of the docs repo.
2. Commit your changes to the fork.
3. Merge your PR into the active branch.

Watch the video for a more detailed walkthrough." Or something?

Reply:
@paperclypse I revised this as you suggested.